Expensify/App
# Fix Search - Arrow navigation on search stops working

### Details

[Comment](https://github.com/Expensify/App/issues/6131#issuecomment-955619193)
### Fixed Issues

$ https://github.com/Expensify/App/issues/6131

### Tests / QA

1. Press `Control + K`
2. Enter something search
3. Press the down arrow several times
4. Verify that you can navigate the search list with the arrow keys

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/29673073/141044840-0c16a176-83b0-4789-be01-b69192355790.mp4

#### Mobile Web

Tested for crash.
#### Desktop

https://user-images.githubusercontent.com/29673073/141047135-a65616da-976e-428e-8bb2-5f287ae1b8c9.mov
#### iOS

Tested for crash.
#### Android

Tested for crash.
